### ISSUE TITLE
Yatopia Download fails

### ISSUE THREAD

**Issue by flololan** [itzg/docker-minecraft-server#817]
Hi,

The Image can't download the latest stable yatopia anymore. Not sure if this is a temporary error or if the path needs to be changed. The only releases i find are here: https://yatopiamc.org/download.html

      
```

[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 1 1000 1000 16 Mar 29 20:26 /data'

[init] Resolved version given LATEST into 1.16.5

[init] Resolving type given YATOPIA

[init] Downloading Yatopia from https://api.yatopiamc.org/v2/stableBuild/download?branch=ver/1.16.5 ...

curl: (22) The requested URL returned error: 404 

[init] ERROR: failed to download from https://api.yatopiamc.org/v2/stableBuild/download?branch=ver/1.16.5 (status=0)
```

**Comment by c-condor** [comment]
I too am getting this same error and I tried downloading the current yatopia file and set the force-redownload to false to try and use the locally downloaded version and that didn't work either.

**Comment by itzg** [comment]
Reported https://github.com/YatopiaMC/api.yatopiamc.org/issues/1

**Comment by ilarramendi** [comment]
https://github.com/YatopiaMC/api.yatopiamc.org/issues/1#issuecomment-821286486

Seems that there is no "stable" version for 1.16.5 for now, maby the error message of "Build not found" from the url should be added to the exit message from the container to get a bit more information?

Also im trying to download the latest build but it still failing with the code:
```
curl: (18) transfer closed with 54496269 bytes remaining to read
[init] ERROR: failed to download from https://api.yatopiamc.org/v2/latestBuild/download?branch=ver/1.16.5 (status=0)
```
which is a working link and the download works executing the same curl command on my terminal

**Comment by itzg** [comment]
I guess I need to add an option for "latest" or "latest stable" lookup.
